testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeDeltaLakeDatabricks104.java

```diff
@@ -35,4 +35,10 @@ String databricksTestJdbcUrl()
     {
         return requireNonNull(System.getenv("DATABRICKS_104_JDBC_URL"), "Environment DATABRICKS_104_JDBC_URL was not set");
     }
+
+    @Override
+    protected String trinoParquetCompressionCodec()
+    {
+        return "SNAPPY";
```

Comment:
10.4 doesn't support ZSTD compression? `TestDeltaLakeDatabricksInsertCompatibility#testCompression` indicates SELECT statement works in the version.